### ISSUE TITLE
logging framework clashes

### ISSUE THREAD

**Issue by yoeluk** [Tecsisa/reactive-cassandra-phantom#2]
When I add the reactive-cassandra-phantom dependency to my play project I get this error when I try to `run` the app.

```
java.lang.ClassNotFoundException: akka.event.slf4j.Slf4jLoggingFilter
```

It seems that the reactive-cassandra-phantom logging framework clashes with the play/akka one. Is there a way around it?

**Comment by juanjovazquez** [comment]
@yoeluk I've added akka-slf4j as a runtime dependency. Please, check if this fixes your issue. Thanks.

**Comment by yoeluk** [comment]
Juan, it didn't fix it - I published your changes locally to make sure. I have a number of dependencies in this project but later I will make a simpler mock play project and see it the issue persists.

**Comment by juanjovazquez** [comment]
I didn't publish a new snapshot with this commit so you'll have to package and publish-local by yourself. Please, try to make sure that akka-slf4j (2.4.0-RC2) is provided in runtime. This dependency is usually present in Akka based projects so it's not usually a problem. I have less experience with Play. You could also try to add the dependency explicitly:

```
"com.typesafe.akka" %% s"akka-slf4j" % "2.4.0-RC2"
```

Please, tell me how it goes.

**Comment by yoeluk** [comment]
I was referring to that exactly. I even changed the version and then did `publish-local` to make sure that sbt was picking up my local copy. I will get to the bottom of it later when I get another chance.

**Comment by yoeluk** [comment]
This worked: 1) downgrading the reactive-cassandra-phantom to 2.3.13 (it seems that play 2.4.3 uses this), 2) alternatively, as you suggested, adding `"com.typesafe.akka" %% s"akka-slf4j" % "2.4.0-RC2"` dependency directly to the built.sbt also worked!

I am unsure why sbt didn't evict `2.3.13` in favour of `2.4.0-RC2`. The currently published version 0.0.7 allow play to `run` after importing the `2.4.0-RC2` dependency directly. So I am going to close the issue. 

Thanks a million for the lib and your help today!

**Comment by juanjovazquez** [comment]
Thanks. It was great to help!
